docs/api_reference_guide.md

```diff
@@ -1,28 +0,0 @@
-# API Reference Guide {#ovms_docs_server_api}
```

Comment:
Do we want to resign completely from that document that sums up what APIs we have?

Reply:
In my opinion, the introductory sentence of `Write a Client Application` article:
"OpenVINO Model Server exposes two sets of APIs:"  
would substitute it just fine. What do you think guys? @kblaszczak-intel @tsavina

Reply:
I agree that removing one level in the menu is a good idea. If the main article in the section can hold the info for all API sub-articles, it will make things cleaner.
Just one thing, we need to make sure the TOCtree and the list in the article are aligned and make sense.

Reply:
Okay, that makes sense